src/component/seed-input.js

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+import { StyleSheet, View } from 'react-native';
+import PropTypes from 'prop-types';
+import Background from '../component/background';
+import MainContent from '../component/main-content';
+import { Header } from '../component/header';
+import { H1Text, Text } from '../component/text';
+import { Button, BackButton, GlasButton } from '../component/button';
+import { InputField } from '../component/field';
+import Card from '../component/card';
+import { FormSubText } from '../component/form';
+import { color, font } from '../component/style';
+
+//
+// Generic Seed Input
+//
+
+const styles = StyleSheet.create({
+  content: {
+    justifyContent: 'flex-end',
+  },
+  title: {
+    textAlign: 'center',
+    marginBottom: 20,
+  },
+  card: {
+    maxHeight: 350,
+    maxWidth: 680,
+    paddingLeft: 45,
+    paddingRight: 45,
+    paddingBottom: 50,
+  },
+});
+
+const SeedInput = ({
+  title,
+  copy,
+  navBack,
+  seedInputs,
+  indexes,
+  setSeedVerify,
+  checkSeed,
+}) => (
+  <Background image="purple-gradient-bg">
+    <Header>
+      <BackButton onPress={() => navBack()} />
+      <Button disabled onPress={() => {}} />
+    </Header>
+    <MainContent style={styles.content}>
+      <View>
+        <H1Text style={styles.title}>{title}</H1Text>
+      </View>
+      <Card style={styles.card}>
+        <FormSubText>{copy}</FormSubText>
+        {indexes.map((seedIndex, i) => (
+          <SeedEntry
+            seedIndex={seedIndex}
+            value={seedInputs[i]}
+            onChangeText={word => setSeedVerify({ word, index: i })}
+            key={i}
+            autoFocus={i === 0}
+            onSubmitEditing={() => checkSeed()}
+          />
+        ))}
+      </Card>
+      <GlasButton onPress={() => checkSeed()}>Next</GlasButton>
+    </MainContent>
+  </Background>
+);
```

Comment:
Having taken another look at this component, I'm not sure it's a good abstraction to bundle all the contents of a view in a component. I'd suggest to export/reuse just the `SeedEntry` and rename this file `seed-entry.js`.